README.md

```diff
@@ -6,16 +6,15 @@
 <!--ts-->
   * [Background](#background)
   * [Definitions](#definitions)
-  * [Logging Configuration](#logging-configuration)
-    * [Brief overview](#brief-overview)
-    * [Variables](#variables)
-      * [Logging_inputs options](#logging_inputs-options)
-      * [Logging_outputs options](#logging_outputs-options)
-      * [Logging_flows options](#logging_flows-options)
-      * [Security options](#security-options)
-      * [Server performance optimization options](#server-performance-optimization-options)
-      * [Other options](#other-options)
-    * [Update and Delete](#update-and-delete)
+  * [Brief overview](#brief-overview)
```

Comment:
You replaced `Brief overview` with `Logging Configuration Overview`?
https://github.com/linux-system-roles/logging/pull/228/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L61-R58